sockeye/training.py

```diff
@@ -316,26 +345,26 @@ def _forward_backward(self, batch: data_io.Batch, is_update_batch: bool = True):
         """
         batch = batch.load(device=self.device)
         with torch.cuda.amp.autocast(cache_enabled=False) if self.using_amp else utils.no_context():  # type: ignore
-            # Forward
-            outputs = self.training_model(batch.source, batch.source_length, batch.target, batch.target_length)
-            # Loss (scaled by update interval)
-            loss_outputs = [loss_function(outputs, batch.labels) for loss_function in self.loss_functions]
-            # TODO(mdenkows): We currently give 1/N weight to every batch in the
```

Comment:
Do we still need this comment? It is not part of the ModelWithLoss object.

Reply:
Added back a version of the comment